### PR TITLE
Add homework10

### PR DESCRIPTION
**Pull request by SergeyLinnickHub**

### CODE REVIEW

File: homework10/server.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+            if (err) {
+                res.end('<strong>Error!</strong>');
+                throw new Error('Error while reading index.html');
+            };
+            res.end(data);
+          });
+    
+          return;
+    }
+    const now = new Date();
```
Comment (iliatcymbal) [inline homework10/server.js]: why need this variable?

File: homework10/server.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        if (err) {
+            res.end('<strong>Error!</strong>');
+            throw new Error('Error while reading index.html');
+        };
+
+        const now = new Date();
+        const dateFooter = `<footer>${now}</footer>`;
+        const position = data.search("</body>");
+        const modificatedData = [data.slice(0, position), dateFooter, data.slice(position)].join('');
+        res.end(modificatedData);
```
Comment (iliatcymbal) [inline homework10/server.js]: you've made it in some complicated way, why don't just
```data.replace('</body>', `${dataFooter}</body>`)```

File: homework10/server.js (hunk 1 above)
Comment (SergeyLinnickHub) [reply homework10/server.js]: this duplicate needs to be removed

File: homework10/server.js (hunk 2 above)
Comment (SergeyLinnickHub) [reply homework10/server.js]: yes it is much more elegant solution :)

**Pull request merged**